datalake/src/main/java/com/sequenceiq/datalake/service/consumption/ConsumptionService.java

```diff
@@ -0,0 +1,55 @@
+package com.sequenceiq.datalake.service.consumption;
+
+import javax.ws.rs.WebApplicationException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.common.exception.WebApplicationExceptionMessageExtractor;
+import com.sequenceiq.consumption.api.v1.consumption.model.request.StorageConsumptionRequest;
+import com.sequenceiq.consumption.client.ConsumptionInternalCrnClient;
+
+@Service
+public class ConsumptionService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConsumptionService.class);
+
+    private final ConsumptionInternalCrnClient consumptionInternalCrnClient;
+
+    private final WebApplicationExceptionMessageExtractor webApplicationExceptionMessageExtractor;
+
+    public ConsumptionService(ConsumptionInternalCrnClient consumptionInternalCrnClient,
+            WebApplicationExceptionMessageExtractor webApplicationExceptionMessageExtractor) {
+        this.consumptionInternalCrnClient = consumptionInternalCrnClient;
+        this.webApplicationExceptionMessageExtractor = webApplicationExceptionMessageExtractor;
+    }
+
+    public void scheduleStorageConsumptionCollection(String accountId, StorageConsumptionRequest request) {
+        try {
+            consumptionInternalCrnClient.withInternalCrn()
```

Comment:
Can we get additional debug log here with the parameters which are required for the Consumption endpoint?